lib/cmd/internal/df.js

```diff
@@ -90,7 +91,8 @@ function createTableForApps(apps) {
 
 // DynoFarm apps
 function apps(cb) {
-  request(getDynoFarmUrl() + "apps", function (err, response, body) {
+  var proxyRequest = request.defaults({'proxy': fhc.config.get("proxy")})
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)